contracts/src/protocol/OmniAVSStorage.sol

```diff
@@ -11,6 +11,9 @@ abstract contract OmniAVSStorage {
     /// @notice Ethereum addresses of currently register operators
     address[] internal _operators;
 
+    /// @notice Map operator address to secp256k1 public key
+    mapping(address => bytes) internal _operatorPubkeys;
+
```

Comment:
Added this here. I don't think we should upgrade our goerli avs. We should either just deploy a new one, and have operators reregister with public keys. Or wait until eigenlayer moves to holesky.